lib/Doctrine/DBAL/Migrations/Version.php

```diff
@@ -187,14 +184,14 @@ public function addSql($sql, array $params = [], array $types = [])
         if (is_array($sql)) {
             foreach ($sql as $key => $query) {
                 $this->sql[] = $query;
-                if (isset($params[$key]) && !empty($params[$key])) {
+                if (!empty($params[$key])) {
```

Comment:
no. ``empty`` is a special construct like ``isset``, so this change is indeed valid